donut/modules/directory_search/templates/view_user.html

```diff
@@ -0,0 +1,159 @@
+{% extends 'layout.html' %}
+{% block page %}
+	<div class='col-md-6 col-md-offset-3' id='dir-info' style='float: none; padding-bottom: 25px'>
+		{% if user is none %}
+			No such user!
+		{% else %}
+			<h2>
+				{{ user['preferred_name'] or user['first_name'] }}
+				{{ user['middle_name'] or '' }}
+				{{ user['last_name'] }}
+				{% if is_me %}
+					<a href='{{ url_for("directory_search.edit_user") }}' style='float: right; padding-right: 20px'>Update</a>
+				{% endif %}
+			</h2>
+			{% if user['image'] %}
+			<div class='col-md-8'>
+			{% else %}
+			<div>
+			{% endif %}
+				<ul class='list-group list-group-root'>
+					{% if user['username'] %}
+						<li class='list-group-item'>
+							<strong>Username</strong>:
+							{{ user['username'] }}
+						</li>
+					{% endif %}
+					<li class='list-group-item'>
+						<strong>UID</strong>:
+						{{ user['uid'] }}
+					</li>
+					{% if user['birthday'] %}
```

Comment:
Should check `is_me`. Birthday is PII. For example it is used to sign up and has been used for elections as well.